drivers/sensor/scd30/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Sensirion SCD30 CO2 sensor configuration options
+
+# Copyright (c) 2021 Advanced Climate Systems
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig SCD30
+	bool "Sensirion SCD30 CO2 Sensor"
+	depends on I2C
+
+if SCD30
+
+config SCD30_MEASUREMENT_INTERVAL
+	int "Measurement interval in seconds"
+	default 30
+	range 2 1800
+	help
+	  The interval with wich measurement are done.
+	  Has to be in the range [2; 1800].
```

Comment:
question: do you think making this a sensor attribute (ie it can be set at runtime) would be better?

Reply:
You're right. It should be added now. The sample has also been updated to reflect these changes.